src/components/Comment/PostComment.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useContext } from "react";
+import { UserContext } from '../../contexts/UserContext'
+import { postCommentByArticleID } from "../../api";
+
+function PostComment({article_id}) {
+    const [newComment, setNewComment] = useState('');
+    const [err, setErr] = useState(false);
+    const [hasPosted, setHasPosted] = useState(false);
+    const { user } = useContext(UserContext);
+
+    const handleChange = (event) => {
+        const { value } = event.target;
+        setNewComment(value);
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        setErr(false);
+        setHasPosted(true);
+
+        postCommentByArticleID(article_id, user, newComment)
+        .then(() => {
+            setHasPosted(false);
```

Comment:
This is where you should spread the new comment into your current comments. I think that you may be refetching currently???